### ISSUE TITLE
Customization Sync Enhancement

### ISSUE THREAD

**Issue by JasonWoods** [josegomez/epicor-editor#14]
Add ability to see if VS Code or Epicor has the newer version of a Customization (along with date?) and to allow user to select which one will overwrite the other.

**Comment by josegomez** [comment]
This is already built in, if the Version in VS doesn't match the version in Epicor you should get a warning telling you about that issue and asking you if you want to proceed (over-write the Epicor version)
If you click No, you can then sync down the Epicor version to update the VS Code one.

**Comment by JasonWoods** [comment]
Yes. This feature works fine, but the flow can be confusing. Definitely a low priority as it already functions, but the two Messages can be confusing (at least for me).
